patch-emscripten.sh

```diff
@@ -0,0 +1,7 @@
+#/usr/bin/env bash
+if test -f $1/manifold.js; then
+  # I still have no idea why this is needed...
+  sed -i 's/new Worker/new (ENVIRONMENT_IS_NODE ? global.Worker : Worker)/g' $1/manifold.js
+fi
+sed -i 's/var nodeWorkerThreads=require("worker_threads");/const{createRequire:createRequire}=await import("module");var require=createRequire(import.meta.url);var nodeWorkerThreads=require("worker_threads");/g' $1/*.worker.js
```

Comment:
Just some ES6 compatibility issue. This is something upstream should fix, and pretty easy to fix.

Reply:
Is this one that already got fixed upstream?

Reply:
Yes, but I think they have not yet released a new version.